src/olympia/amo/utils.py

```diff
@@ -586,7 +586,7 @@ def get_locale_from_lang(lang):
     """Pass in a language (u'en-US') get back a Locale object courtesy of
     Babel.  Use this to figure out currencies, bidi, names, etc."""
     # Special fake language can just act like English for formatting and such
-    if not lang or lang == 'dbg':
+    if not lang or lang in ('dbg', 'dbr', 'dbl'):
```

Comment:
I left `dbg` since I don't know if there's anything coming out from somewhere that still uses it so I wanted to be better safe than sorry.